python/snewpy/models/presn.py

```diff
@@ -0,0 +1,141 @@
+import numpy as np
+import pandas as pd
+from scipy.interpolate import interp1d
+from astropy import units as u
+from .base import SupernovaModel
+from snewpy.neutrino import Flavor
+from pathlib import Path
+
+def _interp_T(t0, v0, dt=1e-3, dv=1e-10, axis=0):
+    "dilog interpolation"
+    lt0 = np.log(t0 + dt)
+    lv0 = np.log(v0 + dv)
+    lv1 = interp1d(lt0, lv0, axis=axis, fill_value=0, copy=False, bounds_error=False)
+    return lambda t1: np.exp(lv1(np.log(t1 + dt)))
+
+
+def _interp_E(e0, v0, axis=1):
+    "linear interpolation"
+    return interp1d(e0, v0, axis=axis, fill_value=0, copy=False, bounds_error=False)
+
+
+def _interp_TE(times, energies, array, ax_t=1, ax_e=2):
+    def _f(t, E):
+        a_t = _interp_T(times, array, axis=1)(t)
+        a_te = _interp_E(energies, a_t, axis=2)(E)
```

Comment:
Should this be `axis=ax_t` and `axis=ax_e` instead of the hard-coded values?

Reply:
Good catch! Thanks! Fixed in ac90339